packages/composites/augur/test/e2e/pack.test.ts

```diff
@@ -1,91 +0,0 @@
-import * as create from '../../src/methods/createMarkets'
```

Comment:
Why is this file being removed?

Reply:
outdated e2e, they used methods that did not exist within the adapter, they practically did not test anything.